packages/ruleset-bundler/src/plugins/skypack.ts

```diff
@@ -9,9 +9,7 @@ export const skypack = (): Plugin => ({
   resolveId(id) {
     if (DATA_URIS.test(id) || isURL(id)) return;
 
-    const path = id.split('/');
-
-    if (path.length > 0 && isValidPackageName(path[0])) {
+    if (isPackageImport(id)) {
```

Comment:
should be safe, I already account for a variety of situations
https://github.com/stoplightio/spectral/blob/0ae8242f8ec6e36c4876f47aca27ac7bf4022261/packages/ruleset-bundler/src/plugins/__tests__/skypack.spec.ts